client/state/signup/dependency-store/reducers.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	SIGNUP_DEPENDENCY_STORE_RESET,
+	SIGNUP_DEPENDENCY_STORE_UPDATE_STATE,
+} from 'state/action-types';
+
+export function storeState( state = null, action ) {
```

Comment:
Should the initial state, here defined as `null`, be consistent with the state output for `SIGNUP_DEPENDENCY_STORE_RESET`? It would also mean that the state is always of the same type. Thus: `state = {}`